espresso-server/app/src/androidTest/java/io/appium/espressoserver/EspressoServerRunnerTest.kt

```diff
@@ -34,6 +41,24 @@ import org.junit.Assert.assertEquals
  */
 @LargeTest
 class EspressoServerRunnerTest {
+
+    @get:Rule
+    val composeRule = AndroidComposeTestRule(
+        activityRule = EmptyTestRule(),
+        activityProvider = { error("Can't provide current activity") }
+    ).also {
+        composeTestRule = it
+    }
+
+    private val syncComposeClock = Thread{
+        while (!Server.isStopRequestReceived){
+            if (context.getDriverStrategy() is ComposeDriver) {
+                composeTestRule.mainClock.advanceTimeByFrame()
+            }
+            Thread.sleep(10)
```

Comment:
what is the meaning of this magic number?

Reply:
added some info where it is taken from in the comment